src/scss/02_atoms/_icons.scss

```diff
@@ -1,19 +1,213 @@
-.a-icon {
-  font-size: size(large);
+$iconConfig: (
```

Comment:
In generale tutte le icone qui definite non sono configurabili. Il mixin serve solo se ci sono dei parametri. Fare in modo che le dimensioni e i colori siano modificabili come in un `iconFont`